config/config/sc-config.yaml

```diff
@@ -60,6 +60,7 @@ objectStorage:
 
   ## Off-site backup replication between two providers or regions using rclone sync
   sync:
+    activeDeadlineSeconds: 7200
```

Comment:
This comments says that you should increase this further. https://github.com/elastisys/ck8s-issue-tracker/issues/297#issuecomment-2196716260

Reply:
Oh.. I missed that. I'll increase it.

Reply:
Done